src/js/otus/queries.tsx

```diff
@@ -71,6 +73,16 @@ export function useFetchOTU(otuId: string) {
     });
 }
 
+/**
+ * Fetches the history of changes for a single OTU
+ *
+ * @param otuId - The id of the OTU to fetch
+ * @returns A history list of changes for a single OTU
+ */
+export function useFetchOTUHistory(otuId: string) {
+    return useQuery<OTUHistory[], ErrorResponse>(OTUQueryKeys.detail(otuId), () => getOTUHistory(otuId));
+}
```

Comment:
Should be a new query key defined to cover the case of history. I think linking it with detailed is fair enough from a cache invalidation perspective. That said you can't use the exact same key because of conflicts. 

 Consider:

```typescript
histories: () => ["OTU", "details", "history"] as const,
history: (id: string) => ["OTU", "detail", "history", id] as const,
```